src/useScript/useScript.demo.tsx

```diff
@@ -22,7 +24,7 @@ export default function Component() {
     <div>
       <p>{`Current status: ${status}`}</p>
 
-      {status === 'ready' && <p>You can use the script here.</p>}
+      {status === 'ok' && <p>You can use the script here.</p>}
```

Comment:
This technically constitutes a breaking change, no? What was the reasoning for moving away from `'ready'`?